### ISSUE TITLE
Question about fine-tuning

### ISSUE THREAD

**Issue by aheatu** [cmu-enyac/LeGR#6]
Hi there.
Thank you for your great paper.

I have some questions about ['Towards Efficient Model Compression via Learned Global Ranking'.](https://arxiv.org/abs/1904.12368)

If I use LeGR once to find 3 pruned ConvNets, do I need to fine tune 3 times?
If it is correct to fine-tune 3 times, where is the part of the code that fine-tunes the pruned networks obtained through LeGR?

**Comment by RudyChin** [comment]
Hi, 

Thank you for your interests in our work. Yes, it is correct that you fine-tune 3 times.
The fine-tuning code is at https://github.com/cmu-enyac/LeGR/blob/master/legr.py#L218

Best,
Rudy

**Comment by aheatu** [comment]
Hi. After reading at the paper and running the code, I have a few questions.

1. Is the latancy in Figure 8 measured directly in hardware? If you measured it yourself, I am curious about the name of the embedded board containing the ARM A57.

2. In Figure 3-(b), how many hours does the normalized time 1 mean? I am wondering how many hours it actually took to run LeGR.

3. Is there any specific reason for putting MorphNet into the control group? Various pruning techniques have appeared in recent years, why are compared to the relatively old MorphNet?

Thank you.

**Comment by RudyChin** [comment]
Hi,

Let me try to address your questions below:

1. We use NVIDIA TX2 for measuring the latency. On the machine we have installed PyTorch. When measuring, we set it to single thread and single batch size.

2. If I remember it correctly, it should take roughly an hour to finish for LeGR to carry out the search. You can try this out by running the search script.

3. There are indeed many methods out there, we pick MorphNet as it adopts the L1 regularization idea, which is also used in the network slimming paper. We find this idea simple and elegant, which is the reasons for our choices.

Hope these help!
